accname-aam/accname-aam.html

```diff
@@ -472,6 +472,7 @@
 			<h2>Substantive changes since the <a href="http://www.w3.org/TR/2014/WD-accname-aam-1.1-20141211/">last public working draft</a></h2>
 			<ul>
 				<li>08-June-2017: Remove list style information from the accessible name computation.</li>
+				<li>08-June-2017: Editorial change give example of text alternative element of &lt;label&gt; in the terminology sections 2A and 2F.</li>
```

Comment:
It was to address an issue Cynthia had and she asked for clarity by providing an example